connector/datadogconnector/README.md

```diff
@@ -106,6 +106,65 @@ service:
 
 Here we have two traces pipelines that ingest the same data but one is being sampled. The one that is sampled has its data sent to the datadog backend for you to see the sampled subset of the total traces sent across. The other non-sampled pipeline of traces sends its data to the metrics pipeline to be used in the APM stats. This unsampled pipeline gives the full picture of how much data the application emits in traces.
 
+## Configurations
+
+```yaml
+connectors:
+    datadog/connector:
+      ## @param ignore_resources - list of strings - optional
```

Comment:
Wondering if we should put this in a `traces` section

Reply:
Makes sense